source/content/timeouts.md

```diff
@@ -24,7 +23,7 @@ Rules are for the good of the group, and timeouts are no exception. Timeouts are
 | Pantheon executed Drupal cron                                                                                                | 180 seconds | Only applies to Pantheon's automatic hourly execution of drush cron |
 | [PHP set_time_limit](https://secure.php.net/manual/en/function.set-time-limit.php)                                           | 120 seconds | Number of seconds a script can run. If reached, the script returns a fatal error. |
 | Load Balancer                                                                                                                | 120 seconds | Applies to HTTPS requests and requests to a DNS A record. Requests using the Pantheon CNAME for HTTP requests are *not* limited |
-| SSH                                                                                                                          | 10 minutes with no communication <br /> 60 minutes hard limit | Applies to remote Drush and WP-CLI commands, SSH tunneling, SFTP, rsync |
+| SSH                                                                                                                          | 5 minutes with no communication; <br /> 60 minute hard limit | Applies to remote Drush and WP-CLI commands, SSH tunneling, SFTP, rsync  |
```

Comment:
this is not what we changed. we changed an authentication timeout.  I think that requires new copy / entry here 

TImeouts:
Authentication - 5 mins  
idle  - 10 mins  
Currently there is _hard_ timeout on active sessions but I am ok with folks being told there is 1 hour.

Reply:
there are other limitations

per user there are only allowed max 20 connections to the same site-env. Tho this is more of a protection against denial of servic. We haven't seen a user trigger this limit in our metrics from the last 3 months.

Similarly there is a total 500 active sessions per node in the platform. This too is to protect us from malicious or accidental DOS. Again we haven't actually see this limit hit in practice.

Reply:
thanks, @spheromak!
I put that `10` back, and adjusted for the no-hard timeout to reflect current Platform behavior.

for the other points about max connections and active sessions, I think we can consider other docs to include those in if it's something our users or CSEs might encounter, and I'm going to leave it out of this PR for now as it's outside of the `timeouts` need.